packages/token-bridge-sdk/src/hooks/useTransactions.ts

```diff
@@ -489,6 +493,22 @@ const useTransactions = (): [Transaction[], TransactionActions] => {
     }
   }
 
+  const setDepositsInStore = (newTransactions: Transaction[]) => {
+    // appends the state with a new set of transactions
+    // useful when you want to display some transactions fetched from subgraph without worrying about existing state
+
+    let transactionsMap: { [id: string]: Transaction } = {}
+
+    ;[...transactions, ...newTransactions].forEach(tx => {
```

Comment:
```suggestion
    [...transactions, ...newTransactions].forEach(tx => {
```

Reply:
This is automatically getting added on format, otherwise it results in an error. Quite weird.